### PR TITLE
created a new file called vcodez.xyz.json

### PR DESCRIPTION
**Pull request by Miya25**

### CODE REVIEW

File: data/lists/vcodez.xyz.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "added": 1533208495,
+    "name": "Topic Bot List",
+    "url": "vcodez.xyz",
+    "icon": "https://media.discordapp.net/attachments/922329669884342302/927144475397087242/aa531606-a037-4e5a-b184-c28624e1e116_static.png?width=213&height=213",
+    "language": "English",
+    "display": 1,
+    "defunct": 0,
+    "discord_only": 1,
+    "description": null,
+    "api_docs": "https://api.vcodez.xyz/en/",
```
[the remark this answers is not in the document]
Comment (Miya25) [reply data/lists/vcodez.xyz.json:12]: ohh i guess its offline let me change it

**Comment by Miya25** [comment]
nvm somethings are not right at the moment and will take some time to be fixed

**Comment by MattIPv4** [comment]
@Miya25 Any progress here?

**Comment by Miya25** [comment]
> @Miya25 Any progress here?

yeah but it will take time, the problem seems to be big

**Pull request closed without merging**